stdlib/public/core/InlineArray.swift

```diff
@@ -12,34 +12,88 @@
 
 /// A fixed-size array.
 ///
-/// The `InlineArray` type is a specialized array that stores its elements
-/// contiguously inline, rather than allocating an out-of-line region of memory
-/// with copy-on-write optimization.
+/// An `InlineArray` is a specialized container that does not use a separate
```

Comment:
Use contractions, per API reference style.

```suggestion
/// An `InlineArray` is a specialized container that doesn't use a separate
```